sdk/core/scheduler/common.h

```diff
@@ -35,7 +35,7 @@ namespace sched
 		/**
 		 * The real type of this subclass.
 		 */
-		enum class Type : uint8_t
+		enum class Type : uint32_t
```

Comment:
Add a comment that this type is deliberately chosen for compatibility with the allocator-based mechanism (xref `sdk/core/allocator/token.h:/^struct SObjStruct`)?